universal-application-tool-0.0.1/test/controllers/admin/AdminProgramControllerTest.java

```diff
@@ -0,0 +1,100 @@
+package controllers.admin;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static play.api.test.CSRFTokenHelper.addCSRFToken;
+import static play.mvc.Http.Status.OK;
+import static play.test.Helpers.*;
```

Comment:
suggestion (blocking): [remove wildcard import](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
done!

Reply:
either intellij or bin/fmt is using wildcards when the list gets real long. i'll see if its just my intellij settings.

Reply:
<img width="930" alt="Screen Shot 2021-02-19 at 9 46 22 AM" src="https://user-images.githubusercontent.com/473671/108541529-67152800-7297-11eb-9c3b-624ff0a3de0b.png">

Reply:
good tip!